src/dmd/cli.d

```diff
@@ -641,6 +641,12 @@ dmd -cov -unittest myprog.d
         Option("vcolumns",
             "print character (column) numbers in diagnostics"
         ),
+        Option("verror-style=digitalmars",
```

Comment:
```suggestion
        Option("verror-style=[digitalmars|gnu|?]",
```

See the other options.

Reply:
better? Not sure '?' makes sense here.

Reply:
Most DMD options have a help page that lists all available options.

> the options are fairly self-explanatory.

Fair enough.